roles/openshift_node/tasks/config.yml

```diff
@@ -70,6 +70,52 @@
     state: directory
   register: temp_dir
 
+- name: Create local temp directory
+  tempfile:
+    state: directory
+  register: local_temp_dir
+  delegate_to: localhost
+  run_once: true
+
+- name: Copy 00-runc-crc.yaml ContainerRuntimeConfig
+  copy:
+    content: >
```

Comment:
Previos runs were failing with 
`    "stderr": "error: error parsing /tmp/ansible.o2py9a38/openshift-ansible-runc-crc.yaml: error converting YAML to JSON: yaml: mapping values are not allowed in this context`

Do you know what the problem is? I _think_ it may be 

```suggestion
    content: |
```

Reply:
Pretty sure we need `|` to preserve new lines, but not sure that will fix the issue entirely.